### PR TITLE
release(v0.13.0-alpha.2): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v0.13.0-alpha.2 release.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/4298)

### CODE REVIEW

**Review by Unix4ever — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,322 @@
...
+*This is a pre-release of Talos*
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/talos-systems/talos/issues.
+
+### Hetzner, Scaleway, Upcloud and Vultr
+
+Talos now natively supports three new cloud platforms:
```
Comment (sergelogvinov) [inline CHANGELOG.md:13]: 4 )

Comment (smira) [reply CHANGELOG.md:13]: thanks, I'll get it fixed for beta!

**Comment by smira** [comment]
integration-provision-2 was restarted here: https://ci.dev.talos-systems.io/talos-systems/talos/20489

**Pull request merged**